masterlist.yaml

```
@@ -664,6 +667,10 @@ plugins:
       - Stats
       - Text
     clean:
+    # 3.3.6
+      - crc: 0x441BC38C
+        util: 'FO3Edit v4.0.4c'
+    # 3.3.3
       - crc: 0x514A0671
         util: 'FO3Edit v4.0.4c'
```

Just remove the old cleaning data, we only keep the latest version.
```suggestion
      - crc: 0x441BC38C
        util: 'FO3Edit v4.0.4c'
```